app/models/ability.rb

```diff
@@ -4,10 +4,13 @@ class Ability
   include CanCan::Ability
 
   def initialize(user)
-    return unless user.present?
-    if user.has_role? :admin
-      if user.has_role? :superuser
-        can [:create, :update, :destroy], User
+    can :read, PastEvent
+    if user.present?
+      if user.has_role? :admin
+        can [:create, :update, :destroy], PastEvent
+        if user.has_role? :superuser
```

Comment:
and this one no need to be nested inside the process of adding roles for admin

Reply:
Done